packages/dapp-connector/src/ScriptInitializers/initializeContentScript.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiName } from './types';
+import { MessengerDependencies, cip30, runContentScriptMessageProxy } from '@cardano-sdk/web-extension';
+
+export interface InitializeContentScriptProps {
+  walletName: string;
+  injectedScriptSrc: string;
+}
+
+// tested in e2e tests
+export const initializeContentScript = (
+  props: Record<ApiName, InitializeContentScriptProps>,
```

Comment:
Right now it will create a cip30 API for each item in props. We need to be able to inject different APIs. Also, right now it will append multiple <script> which is unnecessary, 1 is enough. 
```suggestion
  apis: Record<ApiName, AnyApi>,
```